src/sentry/seer/signed_seer_api.py

```diff
@@ -21,13 +22,20 @@ def make_signed_seer_api_request(
     if timeout:
         timeout_options["timeout"] = timeout
 
-    return connection_pool.urlopen(
-        "POST",
-        path,
-        body=body,
-        headers={"content-type": "application/json;charset=utf-8", **auth_headers},
-        **timeout_options,
-    )
+    with metrics.timer(
```

Comment:
let's make the sample rate 100% while we're in EA / still debugging this?

Reply:
This metric applies to all requests to Seer, not just grouping. Is it worth making it configurable (i.e., making it so you can pass in a higher sample rate)? Or are we fine with collecting 100% for all Seer features?

Reply:
i think its fine